polarization/tissueLayer.py

```diff
@@ -65,18 +62,52 @@ def propagateManyThrough(self, vectors: List[JonesVector]) -> List[JonesVector]:
         return J
 
     def backscatter(self, vector: JonesVector) -> JonesVector:
-        return self.backscatterTransferMatrix(vector.k) * vector
+        deltaX, deltaY = self.scatteringDeltaAt(vector.k)
```

Comment:
Why store deltas and require knowledge of the final matrix when we could simply store the matrix?
Let's define `totalBackscatteringMatrixAt(k)` because this will be general instead of assuming it is a birefringent material.